### PR TITLE
IGVF-1844-content-type-defs

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/tabular_file.json [hunk 1]
```diff
@@ -121,15 +121,39 @@
...
                 "sample sort parameters",
                 "sequence barcodes",
                 "SNP effect matrix",
+                "tissue positions",
                 "variants",
                 "variant binding effects",
                 "variant effects",
                 "variant to element mapping"
             ],
+            "enum_descriptions": {
```
Comment (hitz) [inline src/igvfd/schemas/tabular_file.json:133]: I would replace "Mapping of... " with "A File describing the mapping of ..." for many of these

File: src/igvfd/schemas/tabular_file.json [hunk 2]
```diff
@@ -121,15 +121,39 @@
...
+                "barcode to element mapping": "Mapping of barcodes to genomic elements.",
+                "barcode to sample mapping": "Mapping of barcodes with to biological sample.",
+                "differential gene expression quantifications": "The quantified changes in gene expression levels between different conditions or groups.",
+                "differential transcript expression quantifications": "The quantified changes in expression levels of transcripts between conditions or groups.",
+                "editing templates": "The homology-directed DNA repair (HDR) templates containing SNVs in the library of saturation genome editing assays.",
+                "element quantifications": "The quantified levels of specific genomic elements. Formatting should follow https://github.com/IGVF-DACC/checkfiles/blob/dev/src/schemas/table_schemas/element_quant.json.",
+                "elements reference": "References for the elements studied, including details like genomic coordinates or identifiers.",
```
Comment (hitz) [inline src/igvfd/schemas/tabular_file.json:140]: elements -> genomic elements

File: src/igvfd/schemas/tabular_file.json [hunk 3]
```diff
@@ -121,15 +121,39 @@
...
+                "fold change over control": "The ratio of expression levels or other measurements in an experimental condition relative to a control.",
+                "guide quantifications": "Information and sequencing read counts for each gRNA in a pooled CRISPR screen experiment.",
+                "guide RNA sequences": "The sequences of guide RNAs designed for genome editing. Formatting should follow https://github.com/IGVF-DACC/checkfiles/blob/dev/src/schemas/table_schemas/guide_rna_sequences.json.",
```
Comment (hitz) [inline src/igvfd/schemas/tabular_file.json:143]: ... "designed for genome editing using CRISPR"

File: src/igvfd/schemas/tabular_file.json [hunk 4]
```diff
@@ -121,15 +121,39 @@
...
+                "MPRA sequence designs": "Sequences designed for an MPRA experiment. Formatting should follow https://github.com/IGVF-DACC/checkfiles/blob/dev/src/schemas/table_schemas/mpra_sequence_designs.json.",
+                "peaks": "Detected genomic regions of relative enrichment in coverage data, such as peaks in ChIP-seq or ATAC-seq indicating protein binding sites or chromatin accessibility.",
+                "prime editing guide RNA sequences": "The sequences of guide RNAs used in prime editing experiments. Formatting should follow https://github.com/IGVF-DACC/checkfiles/blob/dev/src/schemas/table_schemas/prime_editing_guide_rna_sequences.json.",
```
Comment (hitz) [inline src/igvfd/schemas/tabular_file.json:146]: ... "prime editing CRISPR experiments"

**Review by hitz — CHANGES_REQUESTED** [review]
a few minor changes

File: src/igvfd/schemas/reference_file.json [hunk 5]
```diff
@@ -217,6 +217,13 @@
                     "permission": "admin_only"
                 }
             ],
+            "enum_descriptions": {
+                "genome reference": "A composite nucleic acid sequence assembled from the sequence of several different individual organisms representing the species.",
+                "elements reference": "References for the elements studied, including details like genomic coordinates or identifiers.",
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/reference_file.json:222]: fixed

File: src/igvfd/schemas/tabular_file.json (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/tabular_file.json:133]: fixed

File: src/igvfd/schemas/tabular_file.json (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/schemas/tabular_file.json:140]: fixed

File: src/igvfd/schemas/tabular_file.json (hunk 3 above)
Comment (ian-whaling) [reply src/igvfd/schemas/tabular_file.json:143]: fixed

File: src/igvfd/schemas/tabular_file.json (hunk 4 above)
Comment (ian-whaling) [reply src/igvfd/schemas/tabular_file.json:146]: fixed

**Review by hitz — DISMISSED** [review]

**Pull request merged**